src/App.js

```diff
@@ -1,10 +1,29 @@
 import "./App.css";
+import Router from "./router/Router";
+import {Link} from "react-router-dom";
+import Navbar from "./components/shared/navbar/Navbar";
+import Footer from "./components/shared/footer/Footer";
+import SideMenu from "./components/shared/sidemenu/SideMenu";
```

Comment:
you can export them in an index file and import them together directly from the index.js file